Enable usage of `Home` and `End` keys in input fields (#19273)
directus/directus: .changeset/friendly-mails-lick.md

---
'@directus/app': patch
---

> Enable Home and End keys do in all inputs

Reviewer: ```suggestion
Enabled usage of `Home` and `End` keys in input fields
```
